unified-runtime/source/common/umf_helpers.hpp

```diff
@@ -244,7 +244,8 @@ poolMakeUniqueFromOpsProviderHandle(umf_memory_pool_ops_t *ops,
                                     umf_memory_provider_handle_t provider,
                                     void *params) {
   umf_memory_pool_handle_t hPool;
-  auto ret = umfPoolCreate(ops, provider, params, 0, &hPool);
+  auto ret = umfPoolCreate(ops, provider, params,
```

Comment:
Could you just use `poolMakeUniqueFromOps`? This function does exactly the same thing, it just accepts the provider as a normal C-handle, not a unique_handle: which, to be honest, is misleading as you are passing ownership to the pool.

Even, if you operate on normal handles, you can just do: `poolMakeUniqueFromOps(provider_unique_handle_t(provider), ...)`

Reply:
Right. `poolMakeUniqueFromOpsProviderHandle()` removed.
Done.